src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/InterfaceMethodsShouldBeCallableByChildTypes.cs

```diff
@@ -72,24 +72,22 @@ private static bool ShouldExcludeOperationBlock(ImmutableArray<IOperation> opera
                     return true;
                 }
 
-                var operations = block.Operations.GetOperations();
```

Comment:
Logic in this analyzer broke as it was relying on specific operation tree shape, which is liable to change in future. I have changed the implementation to be bit more resilient. Also filed https://github.com/dotnet/roslyn-analyzers/issues/1737 to make similar changes for any other such analyzers in our repo (e.g. RemoveUnusedParameters).

Reply:
The changes are more argument towards having a method of filtering implicit operations.

---
In reply to: [203456959](https://github.com/dotnet/roslyn-analyzers/pull/1738#discussion_r203456959) [](ancestors = 203456959)

Reply:
Yes, agreed. Such utilities would be helpful for analyzer authors.